packages/@aws-cdk/aws-backup/README.md

```diff
@@ -82,6 +82,15 @@ const plan = backup.BackupPlan.daily35DayRetention(this, 'Plan', myVault); // Us
 plan.addRule(backup.BackupPlanRule.monthly1Year(otherVault)); // Use `otherVault` for this specific rule
 ```
 
+you can backup VSS(Volume Shadow Copy Service)-enabled Windows applications running on Amazon EC2 instances. If the application has VSS writer registered with Windows VSS, then AWS Backup creates a snapshot that will be consistent for that application.
```

Comment:
can you add a doc link somewhere here?

Reply:
It seems to me that in this project, in general, there is no link to the document in Readme. Is it okay to put this in?

Reply:
It seems to me that Readme of this project generally does not include links to official documents.

Is it okay if I put this in here?

Reply:
READMEs include doc links all the time. Should be many examples around the CDK.